src/modules/AddressBook.js

```diff
@@ -0,0 +1,238 @@
+'use strict';
+
+var Utils = require('../qbUtils');
+var config = require('../qbConfig');
+
+/**
+ * Address Book
+ * @namespace QB.addressbook
+ */
+function AddressBook(service) {
+  this.service = service;
+}
+
+AddressBook.prototype = {
+  /**
+   * Method cud is create, update and delete contacts in address book. <br />
+   * An operation choosens by state: if contacts (to identify uses phone number) doesn't exist in address book it will save,
+   * if contact founds it will update, if passed a property `destroy` with 1 the contact will remove. <br />
+   * {@link https://quickblox.com/developers/AddressBook Found more here}. <br />
+   * The methods accepts 2 or 3 parameters. 
+   * @memberof QB.addressbook
+   * @param {Object[]} list - A list of contacts to create / update / delete.
+   * @param {Object} [options] - Search records with field which contains exactly specified value or by array of records' ids to retrieve
+   * @param {string} [options.udid] - User's device identifier. If specified all operations will be in this context. Max length 64 symbols.
+   * If not - it means a user has one global address book across all his devices.
+   * @param {number} [options.force] - Defines force rewrite mode.
+   * If set 1 then all previous contacts for device context will be replaced by new ones.
+   * @param {Function} callback - The savedAddressBookCallback function
+   * 
+   * @example
+   *  var people = [{
+   *    'name':'Frederic Cartwright',
+   *    'phone': '8879108395'
+   *  },
+   *  {
+   *    'phone': '8759108396',
+   *    'destroy': 1
+   *  }];
+   * 
+   *  var options = {
+   *    force: 1,
+   *    udid: 'A337E8A4-80AD-8ABA-9F5D-579EFF6BACAB'
+   *  };
+   * 
+   *  function addressBookSaved(err, responce) {
+   *    if(err) {
+   *      
+   *    } else {
+   *      console.log('All constacts saved');
+   *    }
+   *  }
+   * 
+   *  QB.addressbook.cud(addressBookList, savedAddressBookCallback);
+   *  // or second parameters can be options
+   *  QB.addressbook.cud(addressBookList, options, savedAddressBookCallback);
+   * 
+   */
+  cud: function(list, optionsOrcallback, callback) {
+    if (!Array.isArray(list)) {
+      new Error('First parameter must be an Array.');
+      return;
+    }
+
+    var opts, cb;
+
+    if(isFunction(optionsOrcallback)) {
+      cb = optionsOrcallback;
+    } else {
+      opts = optionsOrcallback;
+      cb = callback;
+    }
+
+    var data = { contacts: list };
+
+    if(opts) {
+      if(opts.force) {
+        data.force = opts.force;
+      }
+
+      if(opts.udid) {
+        data.udid = opts.udid;
+      }
+    }
+
+    this.service.ajax({
+      'type': 'POST',
+      'dataType': 'json',
```

Comment:
Is it need to set 'json'? [dataType has 'json' by default](https://github.com/QuickBlox/quickblox-javascript-sdk/blob/cad1e05874ef0a95138011042ddd20dae52242b0/src/qbProxy.js#L82)